src/interface/src/styleguide/scenario-card/scenario-card.component.html

```diff
@@ -0,0 +1,57 @@
+<div class="title-row">
+  <div class="title-section">
+    <div class="name-header">{{ scenario.name }}</div>
+
+    <sg-status-chip
+      [status]="getChipStatus()"
+      class="title-chip"></sg-status-chip>
+
+    <div *ngIf="scenario.status === 'FAILED'" class="failed-message">
+      <mat-icon class="error-icon">error</mat-icon>
+      Create a new scenario to proceed to treatment plan
+    </div>
+  </div>
+  <div class="controls-section" *ngIf="scenario.status !== 'FAILED'">
+    <button sg-button icon="add" [disabled]="scenario.status !== 'DONE'">
+      New Treatment Plan
+    </button>
+    <button
+      mat-icon-button
+      [matMenuTriggerFor]="menu"
+      aria-label="more scenario Options"
+      class="more-menu-button">
+      <mat-icon>more_vert</mat-icon>
+    </button>
+    <mat-menu #menu="matMenu">
```

Comment:
@pflopez -- We get a lot for free with mat-menu here, but I'm not sure what the conventions would be for including this in a component like scenario-card. 
For instance, would we pass args for different menu options into this? Or is that overkill for our purposes?

Reply:
Im not sure which options are available, but instead of passing individual options which might get messy, I suggest we use something like the scenario status and let the component figure out which options to show.

Reply:
Not sure if you where referring on how to implement the menu options on _this_ component, or on a general `sg-menu` component (which internally uses `mat-menu`) :D

Reply:
Yeah, I was thinking about that too. Maybe I'll add a card for that in the sprint in lieu of some other ones, then ask for forgiveness for focusing on smaller components before I work on composite ones.